src/utils/web.py

```diff
@@ -73,6 +77,18 @@ def urlencode(*args, **kwargs):
     from urllib.error import HTTPError, URLError
     from urllib.parse import splithost, splituser
 
+if svi[0] == 2 and svi[1] == 7 and svi[2] >= 9:
```

Comment:
You can change this to `svi >= (2, 7, 9)`

Reply:
@wolfy1339 not really, it would make it match python 3. It should be: `(2, 7, 9) <= svi < (3, 0, 0)`